### DISCUSSION TITLE
Embedding vs forwarding

### DISCUSSION THREAD

**Discussion by Anonym69** [YOURLS/YOURLS#3541] [General]
Is it a configuration setting to avoid embedding? I literally h##e embedding. The links become automatically identified as unreliable.
My 1.9.1 didn't do that, then I upgraded to 1.9.2 just 20 mins ago and I got that, going back to 1.9.1 (just copying the code) didn't undo the problem. Several sites doesn't even open the links, whatsapp.com etc.

This is what Firefox gives as a message:

> Firefox Can’t Open This Page
>
> To protect your security, some-domain.com will not allow Firefox to display the page if another site has embedded it. To see this page, you need to open it in a new window.

Try e.g. to shorten this link:

https://faq.whatsapp.com/514799537162613

**Comment by ozh** [comment]
I don't understand what you're writing about. https://faq.whatsapp.com/514799537162613 shortened : https://ozh.in/zv. What's the problem?

**Reply by Anonym69** [reply]
Yours is forwarding to the page. Mine can't be anything else than embedding. That's why I asked if it's a config setting to avoid embedding.

**Reply by dgw** [reply]
I don't understand what's meant by "embedding". Seems like @ozh doesn't either. Any chance you can give us an actual example link with the problem? Screenshot?

**Reply by Anonym69** [reply]
Sure! Sorry about that. What I mean with embedding is when the short url stays within the context of the server where YOURLS is installed, e.g. let's say that the server would be installed on https://myserver.com/ , then let's say that the above whatsapp page is shortened to whatsapp, i.e. https://myserver.com/whatsapp .

In the case of ozv above, when clicking on his shortened link it is forwarded to https://faq.whatsapp.com/514799537162613 , while in my cases (after upgrade to 1.9.2) the shortened link loads an iframe for the link at whatsapp.com .

It wasn't like this before the upgrade (my links where also forwarded) and now it does, hence that I ask if there is a setting for this.

You get me?

**Reply by dgw** [reply]
Yes, I understand now, thanks for explaining. Sounds like you somehow got a toolbar plugin enabled 😕

Such a plugin [ships with YOURLS by default](https://github.com/YOURLS/YOURLS/blob/02eda602ba2590519eb59020e0e7300cb08b1e72/user/plugins/sample-toolbar/plugin.php), actually, but it shouldn't auto-enable itself. Check your plugins list in the admin panel and see if you can find & disable a plugin like that.

**Reply by Anonym69** [reply]
Yey! You were right! Fixed. I must have slipped with my fingers (haha), I am idiot apparently. :)
Thank you!
